easybuild/easyconfigs/t/texlive/texlive-20230313-GCC-13.2.0.eb

```
@@ -0,0 +1,78 @@
+# Based off the 2017 version by John Dey jfdey@fredhutch.org
+# https://github.com/easybuilders/easybuild-easyconfigs/pull/5085
+easyblock = 'Tarball'
+
+name = 'texlive'
+version = '20230313'
+local_ver_year = version[:4]
+
+homepage = 'https://tug.org'
+description = """TeX is a typesetting language. Instead of visually formatting your text, you enter your manuscript
+ text intertwined with TeX commands in a plain text file. You then run TeX to produce formatted output, such as a
+ PDF file. Thus, in contrast to standard word processors, your document is a separate file that does not pretend to
+ be a representation of the final typeset output, and so can be easily edited and manipulated."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+source_urls = [
+    'ftp://tug.org/texlive/historic/%s/' % local_ver_year,
+    'https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/%s/' % local_ver_year,
+]
+sources = [
+    {
+        'download_filename': 'install-tl-unx.tar.gz',
+        'filename': 'install-tl-unx-%(version)s.tar.gz'
+    }
+]
+checksums = ['d97bdb3b1903428e56373e70861b24db448243d74d950cdff96f4e888f008605']
+
+dependencies = [
+    ('X11', '20231019'),
+    ('libpng', '1.6.40'),
+    ('libGLU', '9.0.3'),
+    ('Perl', '5.38.0'),
+    ('HarfBuzz', '8.2.2'),
+    # TODO ('poppler', '23.09.0'),
```

I suppose this has to use the included easyconfig?
```suggestion
    ('poppler', '24.04.0'),
```